theia-docker/Dockerfile

```diff
@@ -30,5 +30,8 @@ COPY --from=0 --chown=theia:theia /home/theia /home/theia
 EXPOSE 3000
 ENV SHELL /bin/bash
 ENV USE_LOCAL_GIT true
+ENV TINI_VERSION v0.18.0
+ADD https://github.com/krallin/tini/releases/download/${TINI_VERSION}/tini /tini
```

Comment:
I got a runtime exception when running `tini` from the entrypoint. Maybe the basic Ubuntu image is missing some dynamic library that's required? It worked with the statically linked version:

`ADD https://github.com/krallin/tini/releases/download/${TINI_VERSION}/tini-static /tini`